rpc/lib/server/handlers.go

```diff
@@ -337,26 +337,26 @@ func jsonStringToArg(cdc *amino.Codec, rt reflect.Type, arg string) (reflect.Val
 	return rv, nil
 }
 
-func nonJSONStringToArg(cdc *amino.Codec, rt reflect.Type, arg string) (reflect.Value, error, bool) {
+func nonJSONStringToArg(cdc *amino.Codec, rt reflect.Type, arg string) (reflect.Value, bool, error) {
 	if rt.Kind() == reflect.Ptr {
-		rv_, err, ok := nonJSONStringToArg(cdc, rt.Elem(), arg)
+		_, ok, err := nonJSONStringToArg(cdc, rt.Elem(), arg)
```

Comment:
why remove `rv_`??? this alters the program logic, no?

Reply:
Yes! Apologies for the oversight.